action_server/src/robocorp/action_server/_server_expose.py

```diff
@@ -27,6 +28,42 @@ class BodyPayload(BaseModel):
     headers: dict
 
 
+class ExposeSessionJson(BaseModel):
+    expose_session: str
+    url: str
+
+
+def get_expose_session_path() -> str:
+    return os.path.join(os.getcwd(), ".robocorp", "expose_session.json")
```

Comment:
Humm, instead of persisting in the current dir, it could be persisted in the settings dir, since that dir is already tied to the current directory (i.e.: `get_settings().datadir`), that way the user doesn't get its current directory dirty due to that.

Reply:
Moved the `expose_session.json` to `settings.datadir`.
Using `get_settings` in expose loop threw "not initialized" error though, so I just changed it to pass `datadir` in process args.